test/libraries/cusolver/dense_generic.jl

```diff
@@ -215,4 +215,34 @@ p = 5
             d_W, neig = CUSOLVER.Xsyevdx!('N', 'V', uplo, d_A, vl=3.5, vu= 7.5, il=1, iu=3)
         end
     end
+
+    @testset "syevBatched!" begin
+        batch_size = 5
+        for uplo in ('L', 'U')
+            (uplo == 'L') && (elty == ComplexF32) && continue
```

Comment:
Should this have a `@test_broken` or something similar to avoid forgetting about it in the future?

Reply:
The issue is that one EVD in the batch randomly fails.
Maybe I shoudl return an error directly in the function if someone wants to use it with this combinaison of parameters (uplo = 'L' and T = ComplexF32).